cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -595,6 +595,18 @@ protected ServicePort createServicePort(String name, int port, int targetPort, S
         return servicePort;
     }
 
+    protected ServicePort createServicePort(String name, int port, int targetPort, int nodePort, String protocol) {
```

Comment:
I'm guessing that `createServicePort(String name, int port, int targetPort, String protocol)` could delegate to this method, passing a null `nodePort`. Similarly `createContainerPort(String name, int port, String protocol)`.